lucene/core/src/java/org/apache/lucene/util/Accountable.java

```diff
@@ -41,4 +41,8 @@
     return Collections.emptyList();
   }
 
+  /**
+   * An accountable that always returns 0
+   */
+  Accountable NULL_ACCOUNTABLE = () -> 0;
```

Comment:
Interface members are always `static final` right?

Reply:
yes

Reply:
To avoid such questions, you can be explicit.

Reply:
sure, but do we then initialize all booleans with `= false`? I thinks it's totally obsolet.